.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,102 @@
+name: Bug Report
+description: Something is not working right
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Before opening a new issue, please take a moment to search through the current open and closed issues to check if it already exists.
+
+        ---
+
+  - type: dropdown
+    id: type
+    attributes:
+      label: Regression?
+      description: |
+        Regression means that something used to work but no longer does.
+        **BEFORE CONTINUING**, please check if this bug is a regression or not, and if it is, we need you to bisect with the help of the wiki: https://wiki.hyprland.org/Crashes-and-Bugs/#bisecting-an-issue
+      multiple: true
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+
+  - type: textarea
+    id: ver
+    attributes:
+      label: Hyprlock Info and Version
+      description: |
+        Please check and enter your hyprlock version.
+        If you are on a packaged version, use your package manager to find your hyprlock version.
+        If you built it yourself, please provide the commit hash.
```

Comment:
We might want to build `--version` into hyprlock.
Is that something that would be nice to have in hyprutils?

Reply:
Nevermind, cannot be in hyprutils, since it is a build time feature right?

Reply:
yea, would be nice.

Reply:
done